### ISSUE TITLE
[Question] Header scrollbar

### ISSUE THREAD

**Issue by lmcadory** [tabalinas/jsgrid#887]
I'm having a CSS/older browser issue. 
I'm using FF 31.6 and Chrome 36. I can't upgrade since they are corporate installed versions. (Developing in a vacuum sucks)

The issue I'm having is with the css for the jsgrid-header-scrollbar. In my FF version the scrollbar shows up with the scroll arrows and in chrome they don't show up at all. I would like them to look like the examples in the demos:
![scrollbar](https://user-images.githubusercontent.com/692895/29570643-0b13e372-8725-11e7-84c1-9781d87ae3c5.PNG)

In FF it looks like this:
![ff](https://user-images.githubusercontent.com/692895/29570867-b157643e-8725-11e7-83f7-8d1b4d299846.PNG)
Notice the arrows in the scroll header.

In older versions of chrome Chrome there's nothing there and the alignment is thrown off

Basically I'm trying to adjust for the spacing that the scrollbar adds to the table. If i remove the scrollbar class from the header it throws the alignment of the table off.  The body of the table is pushed to the left by the width of the scrollbar. 
Is there a way to make sure the header is always aligned with the body of the table?

**Comment by tabalinas** [comment]
Have you tried to play with `overflow` (`overflow-x` and `overflow-y`) of `jsgrid-grid-body` and `jsgrid-grid-header`? There should be a combination that behaves the way you expect it.

**Comment by lmcadory** [comment]
I haven't.
I will play around with it and see what happens

**Comment by coballa** [comment]
Possible solution, CSS: 
` 		@supports (-ms-ime-align:auto) {  /* IE Edge 12+ CSS */ 
			.jsgrid-grid-header {
				padding-bottom: 0.3px !important;
			}
		}`
